src/core/ext/filters/client_channel/subchannel.cc

```diff
@@ -503,6 +500,8 @@ BackOff::Options ParseArgsForBackoffValues(
                 {static_cast<int>(initial_backoff_ms), 100, INT_MAX});
       } else if (0 ==
                  strcmp(args->args[i].key, GRPC_ARG_MIN_RECONNECT_BACKOFF_MS)) {
+        // TODO(juanlishen): Consider rename GRPC_ARG_MIN_RECONNECT_BACKOFF_MS
```

Comment:
I think it's fine to rename this.  We can do this in a backward-compatible way by defining the existing macro as a duplicate of the new one.

If you do this, please add a note about removing the old macro to doc/core/pending_api_cleanups.md.